stl/inc/memory

```diff
@@ -2949,6 +2949,18 @@ _NODISCARD enable_if_t<is_unbounded_array_v<_Ty>, shared_ptr<_Ty>> allocate_shar
 
 template <class _Ty>
 class weak_ptr : public _Ptr_base<_Ty> { // class for pointer to reference counted resource
+private:
+    template <class _Ty2,

```

Comment:
Stupid question, but why isnt this an inline template variable that defaults to false?